.bitmap

```diff
@@ -1434,6 +1440,12 @@
         "mainFile": "index.ts",
         "rootDir": "scopes/ui-foundation/uis/hooks/use-user-agent"
     },
+    "ui/icon": {
```

Comment:
Better be more specific with component names: e.g. icons/lane-icon.

Reply:
@erezbit

Reply:
fixed